src/nft/components/profile/list/RoyaltyTooltip.tsx

```diff
@@ -17,16 +17,20 @@ const RoyaltyContainer = styled(Column)`
   padding: 4px 0px;
 `
 
-const MarketIcon = styled.img`
+const MarketIcon = styled.div`
   width: 16px;
   height: 16px;
-  border-radius: 2px;
-  object-fit: cover;
+  border-radius: 4px;
   outline: 1px solid ${({ theme }) => theme.backgroundInteractive};
   margin-right: 8px;
 `
 
-const CollectionIcon = styled(MarketIcon)`
+const CollectionIcon = styled.img`
+  width: 16px;
+  height: 16px;
+  object-fit: cover;
+  outline: 1px solid ${({ theme }) => theme.backgroundInteractive};
+  margin-right: 8px;
   border-radius: 50%;
```

Comment:
to prevent redundancy can you move the shared styles to a `css` object used in both components